sovrin_client/client/client.py

```diff
@@ -90,21 +84,8 @@ def handlePeerMessage(self, msg):
         """
         return self.peerMsgRouter.handle(msg)
 
-    def _getOrientDbStore(self):
```

Comment:
No, we used to store a graph for client, long ago, then we made it optional, we can again support it but there is no point, we dont want clients to install any db